recipes/pyferret/site_specific.patch

```diff
@@ -0,0 +1,183 @@
+diff -Naur external_functions/ef_utility/platform_specific.mk.x86_64-linux external_functions/ef_utility/platform_specific.mk.x86_64-linux
+--- external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-06-17 21:09:23.000000000 -0300
++++ external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-06-22 18:03:51.357859794 -0300
+@@ -23,7 +23,7 @@
+ LD		= gfortran
+ LD_DYN_FLAGS	= -fPIC -m64 -shared -Wl,--no-undefined
+ SYSLIBS		= ../ef_utility/copy_ferret_ef_mem_subsc.o \
+-		  -L$(INSTALL_FER_DIR)/lib64/$(PYTHON_EXE)/site-packages/pyferret \
++		  -L$(SP_DIR)/pyferret \
```

Comment:
Locally this works and the `external_functions` are built, but remotely it cannot find the `pyferret.so` lib :confused: